appveyor.yml

```diff
@@ -5,6 +5,10 @@ environment:
 
 install:
 
+cache:
+  - nsissetup.exe
+  - NsProcess.zip

```

Comment:
maybe add the dependency on appveyor.yaml so that we don't run intro trouble if we ever want to use newer versions of those two files?
```yaml
cache:
   - nsissetup.exe -> appveyor.yaml
   - NsProcess.zip -> appveyor.yaml
```

Not sure if it would be possible to cache the nuget packages as well, like below. The way I understand the order in which things are supposed to happen, it should work.
```yaml
cache:
   - thirdparty\windows -> thirdparty\fetch-thirdparty-deps.ps1
```